cluster/gce/gci/apiserver_manifest_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"strings"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This shouldn't be necessary, right?

Reply:
I don't think it is (IJ added this automatically), but I do see that we do this in a lot of places in k8s.

Reply:
Interesting. I do see it in a ton of places.

Reply:
For posterity, this is due to how goimports makes assumptions about `foo/v1` being a go modules version 1 of the `foo` package (https://go-review.googlesource.com/c/tools/+/122616/).